docs/head/1.guides/plugins/minify.md

```
@@ -52,10 +52,29 @@ export interface MinifyPluginOptions {
    * @default true
    */
   json?: boolean
+  /**
+   * Omit the line breaks between rendered tags, producing a single line of output.
+   * Only removes the separators between tags; newlines inside inline content are
+   * left to the minifiers above.
+   * @default false
+   */
+  omitLineBreaks?: boolean
 }
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)